umap/static/umap/js/umap.js

```diff
@@ -524,8 +524,8 @@ U.Map = L.Map.extend({
           this.help.hide()
         } else {
           this.panel.close()
-          this.editPanel.close()
-          this.fullPanel.close()
+          if (this.editPanel) this.editPanel.close()
```

Comment:
```suggestion
          this.editPanel?.close()
```

Would it work?

Reply:
Sounds like it is! Is it valid in the browser we support ?

Reply:
[I think so](https://caniuse.com/mdn-javascript_operators_optional_chaining)